modules/articles/client/config/articles.client.menus.js

```diff
@@ -1,8 +1,17 @@
-'use strict';
+/**
```

Comment:
What do you think about ngDoc or jsDoc for these? I believe this project should encourage documenting code more.